### PR TITLE
Add clearlinux to supported platforms

### PR DESCRIPTION
**Pull request by albertomurillo**
Signed-off-by: Alberto Murillo Silva <alberto.murillo.silva@intel.com>

### CODE REVIEW

File: roles/ceph-osd/tasks/pre_requisite.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@
   package:
     name: parted
     state: present
+  when: ansible_os_family != 'ClearLinux'
```
Comment (leseb) [inline roles/ceph-osd/tasks/pre_requisite.yml]: why not?

File: roles/ceph-common/tasks/installs/install_on_clear.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+- name: Install ceph bundle
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_on_clear.yml]: s/Install/install/

File: roles/ceph-osd/tasks/pre_requisite.yml (hunk 1 above)
Comment (albertomurillo) [inline roles/ceph-osd/tasks/pre_requisite.yml]: ClearLinux does not manage packages but "bundles" instead.

The parted package is included already in the storage-cluster bundle

File: roles/ceph-common/tasks/installs/install_on_clear.yml (hunk 2 above)
Comment (albertomurillo) [inline roles/ceph-common/tasks/installs/install_on_clear.yml]: done

File: roles/ceph-common/tasks/installs/install_on_clear.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+- name: install ceph bundle
+  swupd: name=storage-cluster state=present
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_on_clear.yml]:
```
swupd:
  name: storage-cluster
  state: present
```

Comment (albertomurillo) [inline roles/ceph-common/tasks/installs/install_on_clear.yml]: This change has been applied

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-purge_cluster

**Comment by leseb** [comment]
jenkins test jewel-ansible2.2-update_dmcrypt

**Pull request merged**